mods/digimon/moves.js

```diff
@@ -4510,10 +4512,9 @@ exports.BattleMovedex = {
 		},
 		flags: {protect: 1, mirror: 1},
 		accuracy: 100,
-		onAfterMoveSecondarySelf: function (pokemon, target, source, move) {
-			for (let i = 0; i < target.side.active.length; i++) {
-				let allyActive = target.side.foe.active[i];
-				if (allyActive) this.boost({def: 1}, allyActive);
+		onAfterMoveSecondarySelf: function (pokemon, target, move) {
```

Comment:
Last i checked this boosted the opponents stats instead of the allies, so just keep it the same for now

Reply:
How about instead of `pokemon` use `source`?

Reply:
I double checked my old code and realized I had target and source swapped. Now it's correct. Anyways now I know that the user is passed before the target in `AfterMoveSecondarySelf` (I double-checked the `useMove` logic). I will however change `pokemon` to `source` per your request.